src/ui/landing/components/landing.tsx

```diff
@@ -15,36 +24,43 @@ interface Props {
 }
 
 interface Slide {
-  svgImage: any;
+  svgImage: ReactNode;
   title: string;
   infoText: string;
 }
 
-const carouselInfo: Slide[] = [{
-  svgImage: require('src/img/img_onboarding-00.svg'),
-  title: '',
-  infoText: ''
-}, {
-  svgImage: require('src/img/img_onboarding-01.svg'),
-  title: 'Create an independent and secure digital identity.',
-  infoText: 'Collect your data at a secure place. It’s yours, so only you own it!'
-}, {
-  svgImage: require('src/img/img_onboarding-02.svg'),
-  title: 'Have all your data at your fingertips.',
-  infoText: 'See all your data in one safe place. Pull the plug and your data is only yours.'
-}, {
-  svgImage: require('src/img/img_onboarding-03.svg'),
-  title: 'Be aware of the information you share.',
-  infoText: 'See what you shared with whom. Have total control over your data.'
-}, {
-  svgImage: require('src/img/img_onboarding-04.svg'),
-  title: 'Your data is as safe as in your bank account.',
-  infoText: 'We use the latest encryption technology and blockchain to store your sensitive data.'
-}, {
-  svgImage: require('src/img/img_onboarding-05.svg'),
-  title: 'Security is hard to maintain, that is why the storage costs.',
-  infoText: 'The storage of your data is payed in ether. But only the change of data costs.'
-}]
+const carouselInfo: Slide[] = [
+  {
+    svgImage: <Onboarding00 height={'100%'} width={'100%'} />,
+    title: '',
+    infoText: ''
+  },
+  {
+    svgImage: <Onboarding01 height={'70%'} width={'100%'} />,
+    title: 'Create an independent and secure digital identity.',
```

Comment:
Thanks for the comments! I agree with them. I would propose though to avoid putting brain power in this now and address it when applying the new design.